### PR TITLE
Adds deprecatedAPI mapping for Ingress in k8s v1.19

### PR DESCRIPTION
**Pull request by bernermic**
This adds the capability to update the deprecated `networking.k8s.io/v1beta1 Ingress` apiVersion ...

`W0409 06:48:53.778446      85 warnings.go:70] networking.k8s.io/v1beta1 Ingress is deprecated in v1.19+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress`

### CODE REVIEW

File: config/Map.yaml [hunk 1]
```diff
@@ -119,3 +119,7 @@ mappings:
     newAPI: "apiVersion: rbac.authorization.k8s.io/v1\nkind: RoleBindingList"
     deprecatedInVersion: "v1.17"
     removedInVersion: "v1.22"
+  - deprecatedAPI: "apiVersion: networking.k8s.io/v1beta1\nkind: Ingress"
+    newAPI: "apiVersion: networking.k8s.io/v1\nkind: Ingress"
+    deprecatedInVersion: "v1.19"
+    removedInVersion: "v1.22"
```
Comment (hickeyma) [inline config/Map.yaml:125]: Can you move this to where the other `Ingress` entry is in the file?

**Review by hickeyma — CHANGES_REQUESTED** [review]
@bernermic Thank you for pushing this change. I have one small nit.

File: config/Map.yaml (hunk 1 above)
Comment (bernermic) [reply config/Map.yaml]: Sure - I changed the position below the other `Ingress` entry.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks for the change @bernermic

**Pull request merged**